cuqi/experimental/mcmc/_gibbs.py

```diff
@@ -0,0 +1,217 @@
+from cuqi.distribution import JointDistribution
+from cuqi.experimental.mcmc import SamplerNew
+from cuqi.samples import Samples
+from typing import Dict
+import numpy as np
+import warnings
+
+try:
+    from progressbar import progressbar
+except ImportError:
+    def progressbar(iterable, **kwargs):
+        warnings.warn("Module mcmc: Progressbar not found. Install progressbar2 to get sampling progress.")
+        return iterable
+
+# Not subclassed from SamplerNew as Gibbs handles multiple samplers and samples multiple parameters
```

Comment:
As discussed, the suggestion for now would be to create (and reference in this PR) a new issue on subclassing Gibbs from `SamplerNew`. This issue should explain what caused this to not be done in the present PR.

Reply:
Agreed. Added #426

Reply:
Thanks, please add more explanation in #426.

Reply:
Added